components/docs-chef-io/content/automate/ha_chef_backend_to_automate_ha.md

```diff
@@ -17,276 +15,334 @@ gh_repo = "automate"
+`knife-ec-backup` utility backups and restores the data in an Enterprise Chef Server installation, preserving the data in an intermediate, editable text format. It is similar to the knife download, uploads commands, and uses the same underlying libraries. It includes workarounds for unsupported objects by the tools and various Server API deficiencies. The goal is to improve knife download, knife upload, and the Chef Infra Server API to deprecate the tool.
 
 {{< note >}}
 
-- The migration procedure is tested on Chef Server version 14+.
-- The migration procedure is possible above Chef Backend version 2.1.0.
+- The migration procedure is tested on Chef Server versions 14+ and 15+.
+- The migration procedure is tested and is possible on the Chef Backend version above 3.1.1.
 
 {{< /note >}}
 
+## Prerequisites
+
+Check the [Prerequisites](/automate/ha_aws_deployment_prerequisites/#migration) page before migrating.
+
 ## Backup the Existing Chef Infra Server or Chef Backend Data
 
-1.   Execute below command to install Habitat:
+1. Execute the below command to install Habitat:
 
-```cmd
-curl https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh \ | sudo bash
-```
- 
-2.   Execute the below command to install the habitat package for `knife-ec-backup`
+    ```cmd
+        curl https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh \ | sudo bash
+    ```
 
-```cmd
-    hab pkg install chef/knife-ec-backup
-```
+2. Execute the below command to install the habitat package for `knife-ec-backup`
 
-3.   Execute the below command to generate a knife tidy server report to examine the stale node, data etc.
-     In this command:
--   `pivotal` is the name of the user
--   `path of pivotal` is the path where the user's pem file is stored.
--   `node-threshold NUM_DAYS` is the maximum number of days since the last checking before a node is considered stale.
+    ```cmd
+        hab pkg install chef/knife-ec-backup
+    ```
 
-```cmd
-    hab pkg exec chef/knife-ec-backup knife tidy server report --node-threshold 60 -s <chef server URL> -u <pivotal> -k <path of pivotal>
-```
-For Example:
+3. Execute the below command to generate a knife tidy server report to examine the stale node, data, etc. In this command:
 
-```cmd
-    hab pkg exec chef/knife-ec-backup knife tidy server report --node-threshold 60 -s https://chef.io -u pivotal -k /etc/opscode/pivotal.pem
-```
+    - `pivotal` is the name of the user
+    - `path of pivotal` is where the user's pem file is stored.
+    - `node-threshold NUM_DAYS` is the maximum number of days since the last checking before a node is considered stale.
 
-4.   Execute the below command to initiate a backup of your Chef Server data. 
-    In this command:
+    ```cmd
```

Comment:
Opinion: the command should go above before the details of the parameters.
cc: @dishanktiwari2501